contracts/staking/contracts/src/staking_pools/MixinStakingPoolMakers.sol

```diff
@@ -0,0 +1,217 @@
+/*
+
+  Copyright 2019 ZeroEx Intl.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+*/
+
+pragma solidity ^0.5.9;
+pragma experimental ABIEncoderV2;
+
+import "@0x/contracts-utils/contracts/src/LibRichErrors.sol";
+import "@0x/contracts-utils/contracts/src/LibSafeMath.sol";
+import "../libs/LibStakingRichErrors.sol";
+import "../interfaces/IStructs.sol";
+import "../interfaces/IStakingEvents.sol";
+import "../immutable/MixinConstants.sol";
+import "../immutable/MixinStorage.sol";
+import "./MixinStakingPoolRewards.sol";
+import "./MixinStakingPoolModifiers.sol";
+
+
+/// @dev This mixin contains logic for staking pools.
+contract MixinStakingPoolMakers is
+    IStakingEvents,
+    MixinConstants,
+    Ownable,
+    MixinStorage,
+    MixinZrxVault,
+    MixinStakingPoolRewardVault,
+    MixinScheduler,
+    MixinStakeStorage,
+    MixinStakeBalances,
+    MixinStakingPoolRewards,
+    MixinStakingPoolModifiers
+{
+
+    using LibSafeMath for uint256;
+
+    function joinStakingPoolAsMaker(
```

Comment:
No devdoc comment here?

Reply:
Jeez, thx for catching that. Must've gotten truncated in the refactor or rebase. I'll add it back.